### DISCUSSION TITLE
Is there a debugging method?

### DISCUSSION THREAD

**Discussion by Ha0ran2001** [callstack/repack#737] [Q&A] [answered]
### Description

I find some debug methods such as using flipper. But they seem to only work on metro.

**Comment by jbroma** [comment, marked as answer]
Hi @Ha0ran2001,

Re.Pack doesn't support Flipper officially, I'm not sure if it works atm and it has been deprecated for a while now. 

In Re.Pack, you can use either: 
- `experimental-debugger` - [docs](https://reactnative.dev/docs/debugging?js-debugger=new-debugger#opening-the-debugger)
- `remote-debugging` - [docs](https://reactnative.dev/docs/next/other-debugging-methods#remote-javascript-debugging-deprecated)

In RN 0.76 Re.Pack will fully support the new React Native Dev Tools.
